system-tables/system-table-information-schema.md

```diff
@@ -548,6 +558,27 @@ desc statistics;
 +---------------+---------------+------+------+---------+-------+
 ```
 
+`TIDB_INDEXES` 表中列的含义如下：
```

Comment:
Is it in master? I have much fewer fields in this table.

Reply:
Sorry, `TIDB_INDEXES` is wrong here.